content/sonar/widgets/project-coverage/description.yml

```diff
@@ -0,0 +1,4 @@
+name: Metrics
+description: Display the selected Sonar metrics of a new code in an application
+technicalName: sonar
```

Comment:
Technical name should be more precise (and unique), something like `sonarProjectCoverage`